google/cloud/bigtable/tests/instance_admin_emulator.cc

```diff
@@ -535,10 +535,13 @@ int run_server(int argc, char* argv[]) {
   if (argc == 2) {
     server_address += argv[1];
   } else {
-    server_address += "9090";
+    server_address += "0";
   }
   DefaultEmbeddedServer server(server_address);
-  std::cout << "Listening on " << server.address();
+  // Need to flush so the output becomes immediately visible to the driver
```

Comment:
Best to use std::flush explicitly then, methinks.  That is, ... "\n" << std::flush;

Reply:
Curious, why?

Reply:
Just to make the correspondence between code and comment very clear.  A majority of people (in my experience) are confused about endl.

Reply:
Okay.  Done.